SaitamaRobot/modules/approve.py

```diff
@@ -0,0 +1,179 @@
+from SaitamaRobot.modules.disable import DisableAbleCommandHandler
+from SaitamaRobot import dispatcher, DRAGONS
+from SaitamaRobot.modules.helper_funcs.extraction import extract_user
+from telegram.ext import CallbackContext, run_async, Update, CallbackQueryHandler
+import SaitamaRobot.modules.sql.approve_sql as sql
+from SaitamaRobot.modules.helper_funcs.chat_status import (
+    bot_admin, user_admin)
+from telegram import ParseMode, InlineKeyboardMarkup, InlineKeyboardButton
+
+
+@user_admin
+@run_async
+def approve(update, context):
```

Comment:
This should be `@loggable`  to make sure log channel get the info, Also add type hints if possible